### DISCUSSION TITLE
Is the `colour.recovery.XYZ_to_sd_Jakob2019` definition description incorrect?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1156] [Q&A] [answered]
`colour.recovery.XYZ_to_sd_Jakob2019`

[https://colour.readthedocs.io/en/v0.4.2/generated/colour.recovery.XYZ_to_sd_Jakob2019.html](https://colour.readthedocs.io/en/v0.4.2/generated/colour.recovery.XYZ_to_sd_Jakob2019.html)

Description says "Recover the spectral distribution of given ***RGB*** colourspace array using Jakob and Hanika (2019) method."

Don't you mean XYZ array?

- J

**Comment by thomasmansencal** [comment]
Thanks @blue-j, it is indeed incorrect! See #1157

**Reply by blue-j** [reply]
The issue continues in the body of the page as well, have a look-see.  - J

**Reply by thomasmansencal** [reply, marked as answer]
Oh but you are looking at the 0.4.2 docs, those won't change, here are the latest: https://colour.readthedocs.io/en/latest/generated/colour.recovery.XYZ_to_sd_Jakob2019.html
